common/changes/master_2017-01-07-07-01.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Persona now has a default way of generating initials and colors ",
+      "type": "patch"
```

Comment:
I think this qualifies as a minor change, not a patch, as it changes behavior and isn't a bug fix.